### PR TITLE
Fix onnx conversion

### PR DESCRIPTION
**Pull request by ParzivalWolfram**
Fix chain of minor errors, and update requirements lists, to allow for conversion of models to onnx format on CPU and AMD GPU.

### CODE REVIEW

File: infer-web.py [hunk 1]
```diff
@@ -720,7 +713,7 @@ def get_info_str(strr):
         )
     ]
 
-    # step3a:训练模型
+    ####### step3a:训练模型
```
Comment (fumiama) [inline infer-web.py:716]: No need to change these `#` comments.

**Review by fumiama — CHANGES_REQUESTED** [review]
You should always keep a minimun change to make it clear for reviewing.

**Comment by ParzivalWolfram** [comment]
> You should always keep a minimum change to make it clear for reviewing.

This is how Git decided to present the change. I have no control over this.

**Comment by ParzivalWolfram** [comment]
Conflicting changes have been made since PR creation, i'm going to close for the moment.

**Pull request closed without merging**